hash/ibm5170.xml

```diff
@@ -22606,6 +22755,69 @@ Disk 2 includes: ""
 		</part>
 	</software>
 
+	<software name="pcplus9305">
+		<description>PC Plus (Issue 80 - May 1993)</description>
+		<year>1993</year>
+		<publisher>Future Publishing</publisher>
+		<notes><![CDATA[
+Disk includes: 
+"Type Plus": Windows utility for creating your own word designs and logos that can be exported into many graphic and document programs. 
+This useful program (from Serif - the makers of PagePlus) illustrates only some of the features of the full version.
+"WormWorld": Delightful series of puzzles for Windows; the object is to guide a worm safely through an assault course; can you get it past the obstacles? All the puzzles can be solved.
+"Magazine": From this month's issue of PC Plus we have the third part of the Video Library project written in Visual Basic 2 (for Windows).
+From Programmers' Workshop a trio of programs: TBM and SHUFFLE in Basic, and SHOWSIZE in assembly language and COM form.
+]]></notes>
+		<sharedfeat name="platform" value="Windows 3.x" />
+
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="SuperDisk" />
+			<dataarea name="flop" size="737280">
+				<rom name="PC Plus (Issue 80 - May 1993) [Future] [1993] [3.5DD] [Disk 1 of 1].img" size="737280" crc="20ca9ce8" sha1="185bf5de8c1625b395a70c3800643582ec0ffe0b"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pcplus9409">
+		<description>PC Plus (Issue 96 - September 1994)</description>
+		<year>1994</year>
+		<publisher>Future Publishing</publisher>
+		<notes><![CDATA[
+Disk includes: 
+"Locoscript Professional 2 Plus": Create your own documents (one-page limit) using this version of the classic word processor.
+"SinkSub": Windows game in which you frigate, armed with depth charges, must evade and destroy the enemy's best fleet of submarines. 
+"Magazine": Various files, macros bits and pieces from Help Screen. Use Basic's Draw command from Wilf's Programmers' Workshop. A project and a new utility from Visual Basic Workshop.
+]]></notes>
+		<sharedfeat name="platform" value="Windows 3.x" />
+
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="SuperDisk" />
+			<dataarea name="flop" size="1474560">
+				<rom name="PC Plus (Issue 96 - September 1994) [Future] [1994] [3.5HD] [Disk 1 of 1].img" size="1474560" crc="f606a575" sha1="ad9bb3849b9a109c34f04921c71a688042c77c21"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pcplus9504">
+		<description>PC Plus (Issue 102 - April 1995)</description>
+		<year>1995</year>
+		<publisher>Future Publishing</publisher>
+		<notes><![CDATA[
+Disk includes: 
+"Medi8or Entree 2.0": A complete commercial multimedia authoring package. Create your own presentations, and enter the PC Plus competition!
+"DOS Bar 1.31": Add a handy toolbar to DOS applications running under Windows.
```

Comment:
If you're quoting a description from the product, I suggest checking if this is "Adds" rather than "Add". Either could be possible IMO.

Reply:
This text was taken from the "pcplus.txt" file available in the disk.
[PCPLUS.TXT](https://github.com/mamedev/mame/files/10295497/PCPLUS.TXT)

Reply:
Okay, thanks for confirming. So that's fine as is.